pcsx2/Counters.cpp

```diff
@@ -214,7 +213,7 @@ static void vSyncInfoCalc(vSyncTimingInfo* info, Fixed100 framesPerSecond, u32 s
 	u64 hBlank = Scanline / 2;
 	u64 hRender = Scanline - hBlank;
 
-	if (gsRegionMode == Region_NTSC_PROGRESSIVE)
+	if (gsVideoMode >= GS_VideoMode::VESA)
```

Comment:
I think I would never use numeral comparisons for enums. However if you do I would really recommend mentioning in the enum definition in GS.h that the order matters. And if somebody want to extend the enum or remove values he has to reconsider exactly this comparison. However I rather believe this functions screams for a switch were every mode is explicitly mentioned and default asserts(0) and request for upstream notification. Like that it is future proof and the ordering of the enum can be adjusted without consequences.